### DISCUSSION TITLE
Adding a file to the GitHub repository

### DISCUSSION THREAD

**Discussion by Desmond235** [community/community#40960] [New to GitHub]
I already have a repository on GitHub and I've been able to push the repository on git to GitHub, I've added a new set of files to the main branch in git but I'm unable to push it to the GitHub repository. This was the command i used "git push origin main" and CMD says 
"Fatal: 'origin' does not appear to be a git repository"
"Fatal: Could not read remote repository ".
What should I do to resolve the problem?

**Comment by GabyCT** [comment]
We have not told Git the location of the “origin” repository. We’ve initialized a new repository using “git init” and so our repository is not linked to any remote repository by default.
To fix this error, we need to manually tell Git where the remote version of our repository exists. Do this using the [git remote add command ]:

$ git remote add origin ${repo}

This command tells Git the remote called “origin” should be associated with a particular URL. Git now knows where the remote version of our repository exists.
Let’s try to push our code again:
$ git push origin master
